matter_server/common/custom_clusters.py

```diff
@@ -243,6 +245,52 @@ def attribute_type(cls) -> ClusterObjectFieldDescriptor:
 
             value: float32 = 0
 
+        @dataclass
+        class Altitude(ClusterAttributeDescriptor, CustomClusterAttributeMixin):
+            """Altitude Attribute within the Eve Cluster."""
+
+            should_poll = True
```

Comment:
Are you sure this need to be polled as well? :thinking:

Reply:
No, that's not necessary.

Reply:
I didn't meant to remove altitude, just use the subscription mechanism (if subscriptions work on these attributes, that you'd have to test on an actual device).

Reply:
How can I manually test the subscription mechanism with my device?

Reply:
You can just set the log level to verbose, then you'll see what is happening. The `Attribute updated` string comes from the subscription callback: 

https://github.com/home-assistant-libs/python-matter-server/blob/c09db74df9f6ce43a37cff88a148069a0c15fd50/matter_server/server/device_controller.py#L1056-L1067

I actually bought such a device too, and I can see that the altitude gets updated via the subscription:
```
2024-08-16 14:39:29.781 (MainThread) VERBOSE [matter_server.server.device_controller] <Node:264> Attribute updated: 1/319486977/319422484 - old value: 959.7000122070312 - new value: 959.0999755859375
```

So in other words, we can set polled to false here, and it will still update :tada: